### DISCUSSION TITLE
Welcome to squzy Discussions!

### DISCUSSION THREAD

**Discussion by PxyUp** [squzy/squzy#57] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by luluagasantiago** [comment]
Hi! Nice to meet you all.   I'm a computer Eng. Student just currently tinkering around with Go and having fun with it. My first language was C a Go seems like an improved/complete version of it. 
 If there's anything I can help with I'd be happy to do it.
